tests/yam/agama/auto.pm

```diff
@@ -8,13 +8,29 @@ use base 'y2_installbase';
 use strict;
 use warnings;
 
-use testapi 'assert_screen';
-
+use testapi;
+use utils 'clear_console';
 
 sub run {
     assert_screen('agama-main-page', 120);
-    assert_screen('agama-installing', 60);
+    assert_screen('agama-installing', 120);
+
+    my ($ext) = get_required_var('AGAMA_AUTO') =~ /(\.[^.]+)$/;
+    assert_screen('agama-install-finished', 1200);
 
+    # we need to manually reboot after auto installation with jsonnet
+    if ($ext eq '.jsonnet') {
+        $testapi::password = 'linux';
+        select_console 'root-console';
+        # get the json file out
+        assert_script_run("agama config show");
+        assert_script_run("agama config show > /tmp/profile.json");
+        upload_logs('/tmp/profile.json');
+        clear_console;
+        enter_cmd "reboot";
+        assert_screen('grub2', 120);
```

Comment:
for each screen and logical action, please add a blank line in between for readability.

Reply:
done.